node/pkg/watchers/evm/connectors/finalizer_poller.go

```diff
@@ -188,19 +188,19 @@ func (b *FinalizerPollConnector) pollBlock(ctx context.Context, logger *zap.Logg
 	return
 }
 
-func getLatestBlock(ctx context.Context, logger *zap.Logger, conn Connector) (*NewBlock, error) {
-	return getBlockByFinality(ctx, logger, conn, Latest)
+func GetLatestBlock(ctx context.Context, logger *zap.Logger, conn Connector) (*NewBlock, error) {
```

Comment:
I didn't see where these were referenced outside this file and needed to be exported.